benchmarks/Directory.Build.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+  </PropertyGroup>
```

Comment:
None of this should be needed. `Nullable=enable` and `AllowUnsafeBlocks=true` are already defined here: https://github.com/terrafx/terrafx.interop.mimalloc/blob/main/Directory.Build.props#L21-L31

`TreatWarningsAsErrors=false` is undesirable. Warnings should be handled and fixed.

This also needs the "standard header" that the other Directory.Build.props have (https://github.com/terrafx/terrafx.interop.mimalloc/blob/main/sources/Directory.Build.props#L2-L19):
```
<Project>

  <!--
    Directory.Build.props is automatically picked up and imported by
    Microsoft.Common.props. This file needs to exist, even if empty so that
    files in the parent directory tree, with the same name, are not imported
    instead. The import fairly early and only Sdk.props will have been
    imported beforehand. We also don't need to add ourselves to
    MSBuildAllProjects, as that is done by the file that imports us.
  -->

  <PropertyGroup>
    <EmbedUntrackedSources>true</EmbedUntrackedSources>
    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileDirectory)..\Directory.Build.props</MSBuildAllProjects>
    <TerraFXProjectCategory>benchmarks</TerraFXProjectCategory>
  </PropertyGroup>

  <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.props" />
```